content/W2017/W17-tshirt-contest.md

```diff
@@ -0,0 +1,31 @@
+Title: T-Shirt Contest
+Date: 2017-02-05
+Category: Events
+Tags: social, big csters, panel
+Slug: options-panel-w17
+Author: Monica Xu
+Summary: Design a T-Shirt for WiCS 
+
+
+Have a passion for design? 
+Love fashion? 
+What to have your designs worn by everyone around campus? 
+
+Submit a design to
+the WiCS T-Shirt Contest... you could be the next trendsetter!
+
+Designs must adhere to the following:
++  Fit on A4 paper
++  1 or 2 colours on 1 solid colour shirt of choice
++  Include "Women in Computer Science" or "WiCS"
++  Be appropriate and adhere to the UW and WiCS codes of conduct
```

Comment:
This needs to be updated so that the information is consistent with the poster. So we need to add the rule about omitting gendered-symbols etc.